### PR TITLE
Make tests pass on 0.6 2

### PR DESCRIPTION
**Pull request by daschw**
Copy of #907

### CODE REVIEW

**Comment by mkborregaard** [comment]
Holy moly, it passes! @daschw you are the man :100: 👍 :tada:

File: test/imgcomp.jl [hunk 1]
```diff
@@ -99,7 +99,7 @@ function image_comparison_facts(pkg::Symbol;
   for i in 1:length(Plots._examples)
     i in skip && continue
     if only == nothing || i in only
-      @fact image_comparison_tests(pkg, i, debug=debug, sigma=sigma, eps=eps) |> success --> true
+      @fact @eval(image_comparison_tests(Symbol(String(Symbol($pkg))[7:end]), $i, debug=$debug, sigma=$sigma, eps=$eps)) |> success --> true
```
Comment (mkborregaard) [inline test/imgcomp.jl]: I'd think `Symbol(` could replace `Symbol(String(Symbol(`

Comment (daschw) [inline test/imgcomp.jl]: `Symbol($pkg)` returns eg `:Plots.gr`.

Comment (mkborregaard) [inline test/imgcomp.jl]: Gotcha. Lets merge!

**Pull request merged**